src/formats/world.cpp

```diff
@@ -158,7 +238,7 @@ std::vector<game_string> world_segment::read_language(uint32_t offset) {
 	backing->read_v(entries);
 	
 	for(world_string_table_entry& entry : entries) {
-		backing->seek(offset + entry.string.value);
+		backing->seek(offset + entry.string.value + 8);
```

Comment:
This breaks reading the string for R&C2. The offset (the size of the header) is only added on for UYA/DL. I was actually working on this earlier but never merged it to master. See this commit: https://github.com/chaoticgd/wrench/commit/0876492bd1fabcdd5357b0794139e093b97f3fe0